src/nlp_expr.jl

```diff
@@ -147,12 +147,31 @@ end
 _parens(::MIME) = "(", ")", "", "", ""
 _parens(::MIME"text/latex") = "\\left(", "\\right)", "{", "}", "\\textsf"
 
+_op_to_string(::MIME, op) = op
+
+function _op_to_string(::MIME"text/latex", op::Symbol)
+    if op == :&&
+        return "\\wedge"
+    elseif op == :||
+        return "\\vee"
+    elseif op == :(<=)
+        return "\\le"
+    elseif op == :(>=)
+        return "\\ge"
+    elseif op == :(==)
+        return "="
+    else
+        return string(op)
+    end
+ end
+
```

Comment:
This might be asking for too, but it would be nice to have some extensible control of how operators are printed. In DisjunctiveProgramming I would like to use the correct LaTeX representations of logical operators that we introduce (e.g., `\\neg`, `\\implies`). Currently, I hack around this via `replace`: https://github.com/hdavid16/DisjunctiveProgramming.jl/blob/d7e8e7a09b1cb39d628b25d9a0cded2345c9664d/src/print.jl#L74-L100. In this case, we have `GenericNonlinearExpr{LogicalVariableRef}` so I could overload `JuMP.function_string`, but that would duplicate a lot of the same code.

Reply:
How's this now

Reply:
Perfect, thanks!